src/components/Navigation/Sidenav.js

```diff
@@ -4,14 +4,15 @@ import { FormattedMessage } from 'react-intl';
 import { NavLink } from 'react-router-dom';
 import './Sidenav.less';
 
-const isNews = (match, location) => location.pathname !== '/';
+const isNews = (match, location) => location.pathname.match(/trending/);
+const isUserWallet = (match, location) => location.pathname.match(/.@.*(wallet)/);
```

Comment:
This will match `busy.org/@sekhmetwallet` which shouldn't be the case. `.@.+\/(wallet)` seems to work, but I'm not regex magician.

Reply:
yeah that works.